README.md

```diff
@@ -93,6 +149,11 @@ The file now comes with a header describing the categories.
 ❯ yq target/2.375.1.1/mm/my-plugins.yaml
 # This file is automatically generated - please do not edit manually.
 
+# Annotations:
+#  tag:custom:version=...    - sets a custom version
+#  tag:custom:url=...        - sets a custom url
+#  tag:custom:requires=...   - specifies any required dependencies
```

Comment:
What is the syntax for these annotations?
E.g. I _assume_ requires is a comma separated list, but I don't know for sure.

Reply:
Fixed with https://github.com/kyounger/casc-plugin-dependency-calculation/pull/39/commits/5f83954c80add56859c94f2440fc3e26b215049f